types/swig-email-templates/tsconfig.json

```diff
@@ -9,6 +9,7 @@
         "noImplicitThis": true,
         "strictNullChecks": true,
         "strictFunctionTypes": true,
+        "esModuleInterop": true,
```

Comment:
why this change?
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports

Reply:
Had a failing test because I made [this change](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61361/files#r926015943). Will revert both 👍